Implement section#find() in v0.3

jdormit opened the pull request:
I implemented the `find()` and `textSprint()` methods in `section` basically the same way as they were implement for `chapter` in v0.2.

I also wrote unit tests for the `section#find()` method.

fchasen commented:
This looks great!

Would it be possible to use `sprint` from core utils? https://github.com/futurepress/epub.js/blob/v0.3/src/utils/core.js#L371

Also, I'm fine with adding Moby dick, but any reason you didn't use the `Alice` epub from the fixtures?

jdormit commented:
Ah, I didn't see the `sprint` had already been implemented! I've refactored to use the `utils/core.js` version instead of a duplicate one.

I also deleted Moby Dick and used Alice instead - I'm not sure why I didn't do that in the first place!

fchasen commented:
Great, published in `epubjs@0.3.37`
